Doc/library/functools.rst

```diff
@@ -468,7 +468,7 @@ The :mod:`functools` module defines the following functions:
 
       initial_missing = object()
 
-      def reduce(function, iterable, initial=initial_missing, /):
+      def reduce(function, iterable, /, initial=initial_missing):
```

Comment:
I am unsure what to do about this. Should I mention `None`? Should I keep `/`? Any help is appreciated.

Reply:
Nothing.